app/models/planning_application.rb

```diff
@@ -326,6 +326,14 @@ def applicant_and_agent_email
     [agent_email, applicant_email].reject(&:blank?)
   end
 
+  def agent_or_applicant_name
+    if agent_first_name?
```

Comment:
Just wondering if we should cater for the edgiest of edge cases where an agent/applicant has one name only? Like if it's a business name or something? Not sure whether RIPA enforce this or not, but if there's any possibility that one of these fields will be `nil` we need to prevent an error. If RIPA enforce this, then not an issue!

Reply:
we can cater for that - its actually a point of failure across different parts of the email - For example, we have a method called agent? that displays "supporting info" if any of the agent details is present - but the only thing we absolutely require is the agent or the applicant email. We also actually don't even require an address to create an application.

My thought was that if someone is hiring an agent to do something the agent should be required to insert their details properly but its easy enough to check if both fields are empty I guess

Reply:
It could maybe go in as an improvement ticket and we could clarify with product owners exactly what details we should be enforcing so it becomes part of a wider issue. Happy to approve for now.